### PR TITLE
Cass wu part4

### PR DESCRIPTION
**Pull request by richardkchapman**

### CODE REVIEW

File: ecl/wutest/wutest.cpp [hunk 1]
```diff
@@ -143,21 +143,6 @@ bool dump(IConstWorkUnit &w, IProperties *globals)
     return true;
 }
 
-void testPagedWuList(IWorkUnitFactory *factory)
```
Comment (ghalliday) [inline ecl/wutest/wutest.cpp]: Should  you retain this function, but only test a single element sort order?

Comment (richardkchapman) [inline ecl/wutest/wutest.cpp]: I will be replacing it with something in the unit tests (when I get there...)

File: plugins/cassandra/cassandraembed.cpp [hunk 2]
```diff
@@ -2252,13 +2284,31 @@ class TimeStampColumnMapper : implements CassandraColumnMapper
...
+    virtual bool fromXML(CassStatement *statement, unsigned idx, IPTree *row, const char *name, const char * userVal)
     {
-        // never bound
-        return false;
+        // never bound, but does need to be included in the ?
+        return true;
     }
 } timestampColumnMapper;
 
+class HashRootNameColumnMapper : implements CassandraColumnMapper
```
Comment (ghalliday) [inline plugins/cassandra/cassandraembed.cpp]: minor - these class instances could be static

File: plugins/cassandra/cassandraembed.cpp [hunk 3]
```diff
@@ -2566,6 +2660,138 @@ class AttributeMapColumnMapper : implements CassandraColumnMapper
...
+public:
+    virtual IPTree *toXML(IPTree *row, const char *name, const CassValue *value)
+    {
+        CassandraIterator elems(cass_iterator_from_map(value));
+        while (cass_iterator_next(elems))
+        {
+            rtlDataAttr str;
+            unsigned chars;
+            getStringResult(NULL, cass_iterator_get_map_key(elems), chars, str.refstr());
+            StringBuffer elemName(chars, str.getstr());
```
Comment (ghalliday) [inline plugins/cassandra/cassandraembed.cpp]: If this is time critical then elemName could be declared outside, and set - otherwise clearer as it is.

**Pull request merged**